rdrf/rdrf/models/definition/models.py

```diff
@@ -1389,12 +1389,14 @@ class ConsentRule(models.Model):
 
 class DemographicFields(models.Model):
     FIELD_CHOICES = []
+    SECTION_PREFIX = "SECTION:"
```

Comment:
Wouldn't this be better placed on the form instead? Looks more like form than model functionality to me...

Reply:
Moved to the admin form.